features/step_definitions/user_logout_steps.rb

```diff
@@ -0,0 +1,13 @@
+Given(/^I am on the laundry room page$/) do
+  visit '/laundry_room'
```

Comment:
Please use path helpers and avoid hard coding the urls

Reply:
Its leftovers from earlier tests that I forgot to delete